.github/ISSUE_TEMPLATE/release_testnet.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Testnet release
+about: Checklist for releasing a testnet
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+# Testnet Release
+
+Testnet name: X
+Release date: X
+Milestone link: X
+Testnet release manager: X
+
+# Testnet Release Manager Checklist
+
+Monday (week before release):
+- [ ] Create GitHub milestone, work with team to populate the milestone with tickets targeted for the release.
+- [ ] Create new testnet directory with initial genesis allocations for this testnet (make sure all current team members and the Galileo bot have some assets!)
+
+Thursday:
+- [ ] Check in with team again in a release meeting and update the GitHub milestone to ensure it represents what will make it into the testnet.
+
+Friday: Prepare a PR (for merge on Monday) with the following:
+- [ ] Update `discord-addresses` file using the Galileo bot (`cargo run --release history --channel <channel>`)
+- [ ] Update `testnet/<moon>/allocations.csv` file with the above Discord faucet addresses
+- [ ] Update the User Guide to mention the git tag
+
+Monday (release day):
+- [ ] Create new git tag e.g. `006-orthosie`, push to shared remote: `git tag -a <tag_name>` - must be annotated tag for Vergen build. This will begin the release process. Monitor the GitHub action to ensure it completes.
+- [ ] Update peer configuration on `testnet.penumbra.zone`: in Tendermint's `config.toml`, update the `bootstrap-peers` and `persistent-peers` fields to contain the IPs of the Penumbra-operated full nodes.
```

Comment:
The node ID changes during redeployment unless the consensus key is persisted between testnets, which the automation currently doesn't support.

Reply:
i.e. you need to bring up the new testnet, check the node id(`curl -s http://testnet.penumbra.zone:26657/status | jq ".result.node_info.id"` or look at `priv_key.json` in the tendermint data directory), then update the `persistent-peers` to contain the main Penumbra validator's node ID + IP

@avahowell should we be updating `bootstrap-peers` anywhere during a deployment?

Reply:
ah so I linked out to https://guide.penumbra.zone/main/pd/join-testnet/fullnode.html (in the below point) for how to fetch the updated node ID and update the peers since that's described there